test/test.js

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
I am wondering if you could move all the scripts (`test_*.js`) to the `scripts` directory and rename this somehow to something like `test_avro_with_magic_bytes.js` or something shorter.

Reply:
I renamed the file to `test_avro_with_schema_registry.js` and yes, I think it would be good to move the tests out of the top level directory.

Reply:
But the file is still called `test/test.js`. Am I missing something? :thinking: